### PR TITLE
feat: Implement feature flag for people advanced filter - EXO-61829 -Meeds-io/MIPs#39

### PR DESCRIPTION
**Pull request by sofyenne**
We will implement  feature flag for people advanced filter

### CODE REVIEW

File: component/core/src/main/resources/conf/portal/configuration.xml [hunk 1]
```diff
@@ -41,6 +41,18 @@
...
+        <key>PeopleAdvancedFilterFeatureProperties</key>
+        <type>org.exoplatform.container.ExtendedPropertyConfigurator</type>
+        <init-params>
+            <properties-param>
+                <name>PeopleAdvancedFilterFeatureProperties</name>
+                <description>People Advanced Filter Feature enablement flag</description>
+                <property name="exo.feature.peopleAdvancedFilter.enabled" value="${exo.feature.peopleAdvancedFilter.enabled:false}"/>
+            </properties-param>
+        </init-params>
+    </component>
```
Comment (azayati) [inline component/core/src/main/resources/conf/portal/configuration.xml]: indentation

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue [hunk 2]
```diff
@@ -28,7 +28,7 @@
       @click="openBottomMenu">
       fa-filter
     </v-icon>
-    <v-scale-transition>
+    <v-scale-transition v-if="peopleAdvancedFilterFeatureEnable">
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]:
```suggestion
    <v-scale-transition v-if="peopleAdvancedFilterFeatureEnabled">
```

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue [hunk 3]
```diff
@@ -102,7 +102,12 @@ export default {
   data: () => ({
     filterToChange: null,
     bottomMenu: false,
+    peopleAdvancedFilterFeatureEnable: false,
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]:
```suggestion
    peopleAdvancedFilterFeatureEnabled: false,
```

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue [hunk 4]
```diff
@@ -102,7 +102,12 @@ export default {
...
   }),
+  created() {
+    this.$featureService.isFeatureEnabled('peopleAdvancedFilter')
+        .then(enabled => this.peopleAdvancedFilterFeatureEnable = enabled);
```
Comment (azayati) [inline webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]:
```suggestion
        .then(enabled => this.peopleAdvancedFilterFeatureEnabled = enabled);
```

**Review by azayati — CHANGES_REQUESTED** [review]

File: component/core/src/main/resources/conf/portal/configuration.xml (hunk 1 above)
Comment (sofyenne) [reply component/core/src/main/resources/conf/portal/configuration.xml]: fixed !

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue (hunk 2 above)
Comment (sofyenne) [reply webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]: fixed !

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue (hunk 3 above)
Comment (sofyenne) [reply webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]: fixed !

File: webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue (hunk 4 above)
Comment (sofyenne) [reply webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleToolbar.vue]: fixed

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**